MapboxNavigation/NavigationViewController.swift

```diff
@@ -335,7 +335,7 @@ open class NavigationViewController: UIViewController {
      If the `NavigationViewController` is already in the stack, it will open the `StepsViewController` unless it is already open.
      */
     @available(iOS 12.0, *)
-    public class func carPlayManager(_ carPlayManager: CarPlayManager, didBeginNavigationWith navigationService: NavigationService, window: UIWindow) {
+    public class func carPlayManager(_ carPlayManager: CarPlayManager, didBeginNavigationWith navigationService: NavigationService, window: UIWindow, delegate: NavigationViewControllerDelegate?) {
```

Comment:
We need to find a different name for the `delegate` argument. It makes it look like the caller should pass itself in to become the CarPlayManager’s (strongly held) delegate, which is incorrect on several levels. What role does the view controller play in this scenario?

Reply:
the `ViewController` is the delegate for the `NavigationViewController` in this scenario.